docker/Dockerfile.local

```diff
@@ -26,8 +26,11 @@ RUN mkdir -p /cassandra/harry
 COPY ./harry-core/target/lib/* /opt/harry/lib/
 COPY ./harry-core/target/*.jar /opt/harry/lib/
 COPY ./harry-runner/target/lib/* /opt/harry/lib/
-COPY ./harry-runner/target/*.jar /opt/harry/
+COPY ./harry-runner/target/*.jar /opt/harry/lib/
+COPY ./harry-integration/target/lib/ /opt/harry/lib/
+COPY ./harry-integration/target/*.jar /opt/harry/
 COPY ./test/conf/logback-dtest.xml /opt/harry/test/conf/logback-dtest.xml
+COPY ./conf/example.yaml /opt/harry/example.yaml
```

Comment:
since the config is not hardcoded anymore, we have to include the example config